entrypoint.sh

```diff
@@ -14,8 +14,9 @@ replace_in_files() {
         VALUE=`printenv "$VAR_NAME"`
 
         # Replace the variable only if it starts with the name of the directive and remove optional ';'
+        # Some directives will not have spaces after the directive name. i.e opcache
         find $PHP_CONFIG_FILES -type f -exec \
-            sed -i "s/^\(;\)\{0,1\}$DIRECTIVE = [^\n]\+/$DIRECTIVE = $VALUE/g" {} \;
+            sed -i "s/^;\?$DIRECTIVE \?=.*\+/$DIRECTIVE=$VALUE/g" {} \;
```

Comment:
The plus after the `.*` is not necessary.
The reason that the `[^\n]` doesn't match is that the line ends immediately after the `=`. `[^\n]\+` matches any character that is not a new line. But there is no character between the `=` and the `\n`. So it does not match. Changing the modifier to an `*` to match zero or more instead of `+` for one or more will fix the problem:

```suggestion
            sed -i "s/^;\?$DIRECTIVE \?=[^\n]*/$DIRECTIVE=$VALUE/g" {} \;
```